### PR TITLE
Feature/8072 sync consent status back to blue gllue

### PR DESCRIPTION
**Pull request by sergeko2o**

### CODE REVIEW

**Comment by idisblueflash** [comment]
Should we remove `[WIP]` in the name of PR, [WIP] Feature/8072 ...?

File: packages/nodes-base/nodes/Gllue/GllueUpdateConsentStatus.node.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+export class GllueUpdateConsentStatus implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'Gllue UpdateConsent Status Node',
+		name: 'gllue_update_consent_status',
+		icon: 'file:gllue.svg',
+		group: ['transform'],
+		version: 1,
+		description: 'Update Candidate Consent Status in Gllue',
+		defaults: {
+			name: 'Gllue',
```
Comment (idisblueflash) [inline packages/nodes-base/nodes/Gllue/GllueUpdateConsentStatus.node.ts:20]: I'd like to name it `Gllue Update Consent Status`

File: packages/nodes-base/nodes/Gllue/GllueUpdateConsentStatus.node.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+		credentials: [
+			{
+				name: 'gllueApi',
+				required: true,
+			},
+		],
+		properties: [
+			{
+				displayName: 'Candidate ID',
+				name: 'candidate_id',
```
Comment (idisblueflash) [inline packages/nodes-base/nodes/Gllue/GllueUpdateConsentStatus.node.ts]: Should we use `candidateId` instead?
I guess you'd see PyCharm alert for you after you configure the TS
Screenshot 2022-01-20 at 9 55 01 AM

File: packages/nodes-base/nodes/Gllue/services.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import {IDataObject,} from 'n8n-workflow';
```
Comment (sergeko2o) [inline packages/nodes-base/nodes/Gllue/services.ts:1]: I'm keeping this in services.ts to ease merge: I'll move the code into GenericFunctions.ts after merge is done

File: packages/nodes-base/nodes/Gllue/helpers.ts [hunk 4]
```diff
@@ -111,3 +111,10 @@ export async function getResponseByUri(uriGenerated: string, requestMethod: any,
 	return await requestMethod(options);
 }
 
+export function prepareGllueApiUpdateData(id: number|string, updateData: IDataObject): IDataObject {
+	let result = Object.assign({}, updateData);
```
Comment (idisblueflash) [inline packages/nodes-base/nodes/Gllue/helpers.ts]: How about we add UTs for this?

File: packages/nodes-base/nodes/Gllue/services.ts [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+import {IExecuteFunctions,} from 'n8n-core';
+import {Gllue, GllueCandidateCreateEndpoint, GllueCandidateListEndpoint, N8nRequest } from './GenericFunctions';
+import { CONSENT_STATUS_CONSENTED } from './constants';
+import { prepareGllueApiUpdateData } from './helpers';
+
+export class GllueCandidateService {
+	createEndpoint: GllueCandidateCreateEndpoint;
```
Comment (idisblueflash) [inline packages/nodes-base/nodes/Gllue/services.ts:8]: Nice Design to group Gllue related services together. 👍

**Review by idisblueflash — APPROVED** [review]

**Pull request merged**